app/code/Magento/CheckoutAgreementsGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,15 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    checkoutAgreements: [CheckoutAgreement] @resolver(class: "Magento\\CheckoutAgreementsGraphQl\\Model\\Resolver\\CheckoutAgreements") @doc(description: "The Checkout Agreements information")
+}
+
+type CheckoutAgreement @doc(description: "Defines all Checkout Agreement information") {
+    agreement_id: Int @doc(description: "Checkout Agreement identifier")
+    name: String @doc(description: "Checkout Agreement name")
+    content: String @doc(description: "Checkout Agreement content")
+    content_height: String @doc(description: "Checkout Agreement content height")
+    checkbox_text: String @doc(description: "Checkout Agreement checkbox tex")
+    is_html: Boolean @doc(description: "Is Checkout Agreement content in HTML format")
```

Comment:
@naydav @VitaliyBoyko  I think we need to include `mode` here as an ENUM. In the existing checkout flow mode determines if a user must accept the agreements by clicking a checkbox input.
Current mode options: https://github.com/magento/graphql-ce/blob/e60d3a2d8ff3964510ecaa0f7617fa9e482236c4/app/code/Magento/CheckoutAgreements/Model/AgreementModeOptions.php#L10-L12

Rendering Logic
Template: https://github.com/magento/graphql-ce/blob/46bbdd0e3d5796e427899b96c189888423cb55d4/app/code/Magento/CheckoutAgreements/view/frontend/web/template/checkout/checkout-agreements.html#L10-L35
Component: https://github.com/magento/graphql-ce/blob/3fc2f9e4c9e558fd73a7999ddec74cd8efe45d93/app/code/Magento/CheckoutAgreements/view/frontend/web/js/view/checkout-agreements.js#L33-L35